.gitlab-ci.yml

```diff
@@ -86,356 +86,365 @@ workflow:
+      #     - cargo test --no-default-features --features std, ink-experimental-engine --verbose --manifest-path examples/contract-transfer/Cargo.toml
+      #
+      # examples-fmt:
+      #   stage:                           examples
+      #   <<:                              *docker-env
+      #   <<:                              *test-refs
+      #   script:
+      #     - for example in examples/*/; do
+      #         cargo fmt --verbose --manifest-path ${example}/Cargo.toml -- --check;
+      #       done
+      #     - for contract in ${DELEGATOR_SUBCONTRACTS}; do
+      #         cargo fmt --verbose --manifest-path examples/delegator/${contract}/Cargo.toml -- --check;
+      #       done
+      #
+      # examples-clippy-std:
+      #   stage:                           examples
+      #   <<:                              *docker-env
+      #   <<:                              *test-refs
+      #   needs:
+      #     - job:                         clippy-std
+      #       artifacts:                   false
+      #   script:
+      #     - for example in examples/*/; do
+      #         cargo clippy --verbose --manifest-path ${example}/Cargo.toml -- -D warnings;
+      #       done
+      #     - for contract in ${DELEGATOR_SUBCONTRACTS}; do
+      #         cargo clippy --verbose --manifest-path examples/delegator/${contract}/Cargo.toml -- -D warnings;
+      #       done
+      #
+      # examples-clippy-wasm:
+      #   stage:                           examples
+      #   <<:                              *docker-env
+      #   <<:                              *test-refs
+      #   script:
+      #     - for example in examples/*/; do
+      #         cargo clippy --verbose --manifest-path ${example}/Cargo.toml --no-default-features --target wasm32-unknown-unknown -- -D warnings;
+      #       done
+      #     - for contract in ${DELEGATOR_SUBCONTRACTS}; do
+      #         cargo clippy --verbose --manifest-path examples/delegator/${contract}/Cargo.toml --no-default-features --target wasm32-unknown-unknown -- -D warnings;
+      #       done
 
 examples-contract-build:
   stage:                           examples
   <<:                              *docker-env
   <<:                              *test-refs
+  artifacts:
+      name: "$CI_JOB_NAME-$CI_COMMIT_SHORT_SHA.csv"
+      paths:
+          - $CI_MERGE_REQUEST_ID-$CI_COMMIT_SHORT_SHA.csv
   script:
+    - echo $CI_JOB_NAME
+    - echo $CI_JOB_NAME-$CI_COMMIT_SHORT_SHA.csv
     - cargo contract -V
+    - SIZE_OUT=""
     - for example in examples/*/; do
         pushd $example &&
-        cargo contract build &&
-        popd;
-      done
-    - for contract in ${DELEGATOR_SUBCONTRACTS}; do
-        pushd examples/delegator/$contract &&
-        cargo contract build &&
+        SIZE_OUT=$(cargo contract build | grep "Original" | cut -d " " -f 4,6) &&
```

Comment:
I think it would be nice to have an `--output-json` option in `cargo-contract` so that a less ugly `jq` query could be used to get the fields we're interested in